src/app/static/src/tests/integration/downloadResults.itest.ts

```diff
@@ -244,4 +242,25 @@ describe(`download results actions integration`, () => {
 
         }, 3100)
     })
+
+
+    it.skip(`can download comparison output report`, async () => {
+        const commit = jest.fn();
+        const dispatch = jest.fn();
+        const root = {
+            ...rootState,
+            modelCalibrate: {calibrateId: "calibrate123"}
+        };
+
+        const state = {comparison: {downloadId: 123, error: null}}
+
+        await actions.downloadComparisonReport({
+            commit, dispatch, state, rootState: root
+        } as any);
+
+        expect(commit.mock.calls.length).toBe(1);
+        expect(commit.mock.calls[0][0]["type"]).toBe("ComparisonError");
+        expect(commit.mock.calls[0][0]["payload"].detail).toEqual("Missing some results")
+        expect(commit.mock.calls[0][0]["payload"].error).toEqual("FAILED_TO_RETRIEVE_RESULT")
+    })
 })
```

Comment:
skipping this test for now because hintr return `No handler found for GET /download/status`, btw this works locally but returns no handler on build kite.

Response for download result endpoint needs to be altered in `hintr`